### PR TITLE
Move gifs to client readme

### PR DESCRIPTION
**Pull request by WilsonZiweiWang**

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -49,11 +51,28 @@ The go to definition feature currently behaves as follows:
...
+
+![Overrides](doc/overrides.gif)
+
+Variable flag suggestions can be triggered by typing a `[`:
+
+![Variable-flags](doc/variable-flags.gif)
+
+Frequently-used task names:
+
+![Tasks](doc/yocto-tasks.gif)
```
Comment (deribaucourt) [inline client/README.md:74]: Could you replace this gif with `do_install` task, which is much more commonly used?

File: client/README.md [hunk 2]
```diff
@@ -49,11 +51,28 @@ The go to definition feature currently behaves as follows:
 | recipe | recipe definition and all bbappends |
 | symbol | all symbols within the include hierarchy |
 
+![Go-to-definition](doc/go-to-definition.gif)
+
 ### Show definitions of BitBake's defined variables on hover
 *This functionnality requires to [provide the BitBake's folder](#set-bitbakes-path)*
```
[the remark this answers is not in the document]
Comment (WilsonZiweiWang) [reply client/README.md:57]: Amended

File: client/README.md [hunk 3]
```diff
@@ -36,6 +36,8 @@ The following suggestions are currently supported:
 * Context-based suggestions for keywords `inherit`, `require` and `include` (provided by *language-server*)
 * Context-based suggestions for all symbols within the include hierarchy
 
+![Directive-statements](doc/directive-statements.gif)
+
 ### Go to definition
 *This functionnality requires to [provide the BitBake's folder](#set-bitbakes-path)*
```
[the remark this answers is not in the document]
Comment (WilsonZiweiWang) [reply client/README.md]: Amended

File: client/README.md [hunk 4]
```diff
@@ -49,11 +51,28 @@ The go to definition feature currently behaves as follows:
...
 
 Place your cursor over a variable. If it is a BitBake defined variable, then its definition from the documentation will be displayed.
 
+![Hover](doc/hover-information.gif)
+
+### Other code suggestions
+You can trigger the override suggestions with the shortcut or by typing a `:` after an variable name or function name.
+
+![Overrides](doc/overrides.gif)
```
[the remark this answers is not in the document]
Comment (WilsonZiweiWang) [reply client/README.md]: Done, just one gif like this is larger than 3 of them combined.

File: client/README.md [hunk 5]
```diff
@@ -49,11 +51,28 @@ The go to definition feature currently behaves as follows:
...
+
 ### Show definitions of BitBake's defined variables on hover
 *This functionnality requires to [provide the BitBake's folder](#set-bitbakes-path)*
 
 Place your cursor over a variable. If it is a BitBake defined variable, then its definition from the documentation will be displayed.
 
+![Hover](doc/hover-information.gif)
+
+### Other code suggestions
+You can trigger the override suggestions with the shortcut or by typing a `:` after an variable name or function name.
```
[the remark this answers is not in the document]
Comment (WilsonZiweiWang) [reply client/README.md]: Amended

File: client/README.md (hunk 1 above)
Comment (WilsonZiweiWang) [reply client/README.md]: Done

**Comment by deribaucourt** [comment]
Amazing, good job!

**Review by deribaucourt — APPROVED** [review]

**Pull request merged**